exporter/chronicleexporter/protos/chronicle_log.proto

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Google LLC
```

Comment:
Can we add a readme file in the `protos` dir to detail how to generate these?

Reply:
Added